<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimiterCalculator</class>
 <widget class="QMainWindow" name="LimiterCalculator">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>6</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>-20</y>
      <width>20</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Input parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Output Parameters</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalWidget" native="true">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>31</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="PeakThresholdValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="TImeSettingsLabel">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>183</y>
       <width>77</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="ReleaseUnitLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ReleaseValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="PakLimiterLabel">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>96</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Peak Limiter</string>
     </property>
    </widget>
    <widget class="QLabel" name="AttackLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>34</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attack </string>
     </property>
    </widget>
    <widget class="QLabel" name="RMSThresholdLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QComboBox" name="PeakThresholdUnitCombo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>dBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dBfs (for DCX2496 users)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="AttackUnitsLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QLabel" name="RMSLimiterLabel">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>9</y>
       <width>68</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RMS Limiter</string>
     </property>
    </widget>
    <widget class="QLabel" name="PeakThresholdLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QComboBox" name="RMSThresholdUnitCombo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>dBu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dBfs (for DCX2496 users)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ReleaseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>38</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Release</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RMSThresholdValue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AttackValue">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="S">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>280</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Store parameters</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>32</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="SpeakerImpedanceValue">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>36</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="SpeakerImpedanceLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>36</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Impedance (Ohm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SpeakerPowerValue">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>64</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="SpeakerPowerLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>64</y>
       <width>138</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Power consumption (W RMS)</string>
     </property>
    </widget>
    <widget class="QLabel" name="SpeakerInfoLabel">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Speaker characteristics</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="loadDriverButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load Speaker</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteInputDataButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete input</string>
     </property>
    </widget>
    <widget class="QLabel" name="AmplificationInfoLabel">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>92</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Amplifier Characteristics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="AmpPowerLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>146</y>
       <width>227</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Amp power (W RMS for the specific impedance)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SensitivityValue">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>174</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AmpPowerValue">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>146</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadAmpButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Load amplifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="HPFValue">
     <property name="geometry">
      <rect>
       <x>316</x>
       <y>249</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="AmpImpedanceLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>118</y>
       <width>108</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Amp impedance (Ohm)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SensitivityUnitCombo">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>174</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>V sens</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dBu sens</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X Factor</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ProtectionLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>202</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Protection (%)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ProtectionCombo">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>202</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>75</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>249</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HPF (Hertz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AmpImpedanceValue">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>118</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QLabel" name="SensitivityLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>174</y>
       <width>172</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sensitivity (Select the correct Units)</string>
     </property>
    </widget>
    <widget class="QLabel" name="xOverLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>108</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x-Over Characteristics</string>
     </property>
    </widget>
    <widget class="QComboBox" name="OperationMode">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Stereo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bridge</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="AmpImpedanceComBoBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>121</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="outputTable">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>351</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>14</number>
    </property>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>DSP Channel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Amplifier</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>W</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ohm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Speaker</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Configuration</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>W</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ohm</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HPF</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LPF</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Attack</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Limiter TH</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>load file</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveMod">
   <property name="text">
    <string>Modulation maps</string>
   </property>
  </action>
  <action name="saveRaw">
   <property name="text">
    <string>Raw images</string>
   </property>
  </action>
  <action name="saveNorm">
   <property name="text">
    <string>Normalized mod maps</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
